types/peakvault/index.d.ts

```diff
@@ -0,0 +1,233 @@
+// Type definitions for non-npm package peakvault-browser 0.0
```

Comment:
This line seems to indicate that the intent of this package is to be a global browser library, but the use of exports say otherwise. Did you add this because CI failed as your package is not on npm? I don't think that's the right thing to do.